README.md

```diff
@@ -29,6 +29,29 @@ Chipyard is actively developed in the [Berkeley Architecture Research Group][ucb
 
 * See [CONTRIBUTING.md](/CONTRIBUTING.md)
 
+## Chipyard-related Publications
+
+These publications cover many of the internal components used in Chipyard:
```

Comment:
"but should not be used as a replacement for the current and updated documentation"